config/routes/framework.yaml

```diff
@@ -0,0 +1,4 @@
+when@dev:
```

Comment:
Is that the new style in SF5? Not using the config/packages/dev/ directory anymore?

Reply:
Yes it is, I don't now if I like it or not.. but this is what is implemented for 5.3: https://symfony.com/blog/new-in-symfony-5-3-configure-multiple-environments-in-a-single-file

> you won't be forced to use this, but if you use Symfony Flex (which is enabled by default) some files generated automatically in your project might be using it.

Reply:
Cool. I can remember reading that, but my project is still 4.4 so I already forgot about it again^^